src/Docker.jl

```diff
@@ -93,7 +94,17 @@ function build_docker_image(root_path::String, uid::Cint, gid::Cint; verbose::Bo
         open(`docker import - $(image_name)`, "w", verbose ? stdout : devnull) do io
             # we need to record permisions, so can't use Tar.jl
             cd(root_path) do
-                run(pipeline(`tar -c --owner=$(uid) --group=$(gid) .`, stdout=io))
+                get_cmd = tar -> `$(tar) -c --owner=$(uid) --group=$(gid) .`
```

Comment:
Out of curiosity, why assigning an anonymous function to a variable instead of defining a plain function?  Same as in #50

Reply:
Actually, I don't think this function even needed, you can do something like (mostly pseudocode)
```jl
tar = Tar_jll.is_available() ? Tar_jll.tar() : `tar`
run(pipeline(`$(tar) -c --owner=$(uid) --group=$(gid) .`, stdout=io))
```

Reply:
Oh that's really nice.

(I thought we still had to use the `do`-blocks for JLLs.)

Can you make a GitHub "suggestion" with that ternary?

Reply:
> (I thought we still had to use the do-blocks for JLLs.)

https://docs.binarybuilder.org/stable/jll/#ExecutableProduct